### DISCUSSION TITLE
Options with multiple select

### DISCUSSION THREAD

**Discussion by bredecl** [wpbones/WPBones#48] [General]
Hi!
As far as I looked, I didn't find any instructions on how to setup a multiple select on the options dashboard.
But, I figured to do with this:
```
$options=["a"=>"A","b"=>"B"];
WPKirk\Html::select()->multiple(true)->name('General/multipleOption[]')->options($options)->render();
```

but whenever I need to get the settings from the db It's doesn't works because I'm getting an array from options, and throws an error on [Htmltag.php line 355](https://github.com/wpbones/WPBones/blob/9a72511914ae9af22d84b74522c82851d58a0de2/src/Html/HtmlTag.php#L355)

So, I need to pass it as an string, with this:

`WPKirk\Html::select()->multiple(true)->name('General/multipleOption[]')->selected(join(",",$plugin->options->get('General.multipleOption')))->options($options)->render();`

that leads to an non selected html/select...

So I made a modification of the [html method](https://gist.github.com/bredecl/eebb991a52d369cca4f0c6332de3f3da) of  HtmlTag file and works fine

![Captura de pantalla 2024-09-24 a la(s) 12 05 28](https://github.com/user-attachments/assets/9cebfd1b-755a-49ae-9fdd-d37c73f5f93e)

So here's the question, maybe I'm absolutely wrong with the approach... is there any right way to do this?
If not, to made a push request, and create a documentation page with the solution.

Thanks!

**Comment by bredecl** [comment]
Hi @gfazioli can you help me with this? Thanks!

**Reply by gfazioli** [reply]
yep, there is an issue there and I'm going to fix and improve that. Thx for reporting. I'll release it in `v1.6.1` soon 👍

**Reply by bredecl** [reply]
Nice! Thanks
